rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistStorageProviderImpl.java

```diff
@@ -0,0 +1,80 @@
+package co.rsk.peg.whitelist;
+
+import static co.rsk.peg.BridgeStorageIndexKey.LOCK_ONE_OFF_WHITELIST_KEY;
+import static co.rsk.peg.BridgeStorageIndexKey.LOCK_UNLIMITED_WHITELIST_KEY;
+import static org.ethereum.config.blockchain.upgrades.ConsensusRule.RSKIP87;
+
+import co.rsk.bitcoinj.core.Address;
+import co.rsk.bitcoinj.core.NetworkParameters;
+import co.rsk.peg.BridgeSerializationUtils;
+import co.rsk.peg.storage.StorageAccessor;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.tuple.Pair;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+
+/**
+ * Implementation of {@link WhitelistStorageProvider} that saves the whitelist to the Storage and
+ * get the lock Whitelist.
+ */
+public class WhitelistStorageProviderImpl implements WhitelistStorageProvider {
+
+    private LockWhitelist lockWhitelist;
+    private final NetworkParameters networkParameters;
+    private final ActivationConfig.ForBlock activations;
+    private final StorageAccessor storageAccessor;
```

Comment:
let's call it `bridgeStorageAccessor` to be consistent with the other storage providers.

Reply:
Done!